dali/test/python/test_utils.py

```diff
@@ -128,6 +129,51 @@ def get_gpu_num():
     return len(out_list)
 
 
+def _get_absdiff(left, right):
+
+    def make_signed(dtype):
+        if not np.issubdtype(dtype, np.signedinteger):
+            return dtype
+        return {
+            np.dtype(np.int8): np.uint8,
+            np.dtype(np.int16): np.uint16,
+            np.dtype(np.int32): np.uint32,
+            np.dtype(np.int64): np.uint64,
+        }[dtype]
+
+    # np.abs of diff doesn't handle overflow for unsigned types
+    absdiff = np.maximum(left, right) - np.minimum(left, right)
+    # max - min can overflow for signed types, wrap them up
+    absdiff = absdiff.astype(make_signed(absdiff.dtype))
+    return absdiff
+
+
+def _check_absdiff():
+    for i in range(-128, 127):
```

Comment:
In principle, overflow on signed int is UB (that we rely on anyway without this PR). The following one-time check aims to verify it works as expected.

Reply:
Maybe we should explain it in a comment in the code rather than in the PR?

Reply:
done